Notebooks/DC2_Postage_Stamps.ipynb

```diff
@@ -117,19 +154,32 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "def make_cutout_image(butler, ra, dec, vmin=None, vmax=None, label=None,\n",
+    "def make_cutout_image(butler, ra, dec, filter='r', vmin=None, vmax=None, label=None,\n",
```

Comment:
Was there a reason for leaving "filter" off the kwarg list, after all that prep work?

Reply:
👍